src/audio/buffer.c

```diff
@@ -40,8 +40,8 @@ struct comp_buffer *buffer_new(struct sof_ipc_buffer *desc)
 				   "could not alloc structure");
 		return NULL;
 	}
-
-	buffer->addr = rballoc(RZONE_BUFFER, desc->caps, desc->size);
+	/*128 is a wa until dma buffer alignment is passed by ipc*/
+	buffer->addr = rballoc_a(RZONE_BUFFER, desc->caps, desc->size, 128);
```

Comment:
Please leave one space after comment starts and before comments ends:

`` /* this is a comment */

Also please explain what is WA?

Reply:
Changed comments, WA stands for Wourkaround - thought it was obvious - will change that since this will be moved to later commit.